request/configure.go

```diff
@@ -20,15 +20,16 @@ func configure(request *request) {
 		normalizedStorePath, err := normalizePasswordStorePath(store.Path)
 		if err != nil {
 			log.Errorf(
-				"The password store '%v' is not accessible at the location '%v': %+v",
-				store.Name, store.Path, err,
+				"The password store '%+v' is not accessible at its location: %+v",
+				store, err,
```

Comment:
FYI `%+v` prints out the entire struct with field names, so now this looks something like this:

```
The password store '{ID:id1 Name:personal Path:~/.not-a-password-store/}' is not accessible at its location: lstat /home/maximbaz/.not-a-password-store: no such file or directory
```

Reply:
This is quite verbose. IMO we only really need the path in this - because fundamentally, the error is "unable to access a password store at $PATH". If we're eventually passing this message through to the extension, then can't we simply include the relevant details in params, and keep the actual message simple?

What's the reason for having the whole struct there?